imager/imager/tests.py

```diff
@@ -63,18 +59,166 @@ def test_login(self):
         assert fred.is_authenticated()
 
     def test_profile_view_redirect(self):
-        """unauthenticated user redirected when trying
-        to view personal profile"""
-        # response = self.client.get('/logout')
-        response = self.client.get('/profile')
-        self.assertRedirects(response, '/login/', status_code=301)
-
-    def test_profile_view(self):
-        """authenticated user able to view personal profile"""
-        fred = UserFactory(username='fred', password='secret')
-        response = self.loginHelper('fred', 'secret')
-        assert fred.is_authenticated()
-        response = self.client.get('/profile')
-        # import pdb; pdb.set_trace()
-        self.assertEqual(response.status_code, 200)
-        assert 'Welcome, fred' in response.context['body']
+        '''unauthenticated user redirected when trying
+        to view personal profile'''
+        self.client.get('/logout/', follow=True)
+        response = self.client.get('/profile/')
+        self.assertEqual(response.status_code, 302)
+
+    # def test_profile_view(self):
+    #     '''authenticated user able to view personal profile'''
+    #     UserFactory(username='fred', password='secret')
+    #     response = self.client.post('/login/', {'username': 'fred', 'password': 'secret'})
+    #     response = self.client.get('/profile')
+    #     # self.assertEqual(response.status_code, 200)
+    #     assert 'Welcome, fred' in response.context['body']
+
+
+class TestProfileView(LiveServerTestCase):
+    def setUp(self):
+        self.user1 = UserFactory.create(
+            username='alice',
+            first_name='Alice'
+            )
+        self.user1.set_password('secret')
+        self.user1.save()
+        self.user2 = UserFactory.create(
+            username='bob',
+            first_name='Bob',
+            )
```

Comment:
given that your setup for each of these two users is identical except for name, I'd do it with iteration.  More extensible should you decide to create more users later.